README.md

```diff
@@ -1,17 +1,18 @@
-# wasp-open-api
+# openapi-merger
 
-Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of wasp OpenAPI specs defined under `paths` in [`values.yaml`](helm/wasp-open-api/values.yaml).
+Serves a single merged [`OpenAPI`](https://swagger.io/specification/) spec based on any number of openapi OpenAPI specs defined under `paths` in [`values.yaml`](helm/openapi-merger/values.yaml).
```

Comment:
Link to the wasp-openapi-merger in the helm repo rather than to a non-existent path